test/functional/autocmd/autocmd_oldtest_spec.lua

```diff
@@ -81,6 +90,10 @@ describe('oldtests', function()
     fn.delete('Xxx2')
     fn.delete(fname)
     fn.delete('Xout')
+
+    if is_os('win') then
+      assert_log('stream write failed. RPC canceled; closing channel', testlog)
```

Comment:
This is definitely an improvement, thanks. Eventually, do we still want to fix this at a lower level? I.e. if this error truly doesn't matter then we shouldn't log it (for specific cases on Windows).

Reply:
I have no clue, I'm not sure what's happening here currently.

Reply:
Hmm, what makes this specific error different compared to the other errors we "ignore" with `assert_log`. Aka, what makes this particular error different?